src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/FileUploadStart.cshtml

```diff
@@ -8,7 +8,7 @@
 }
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
```

Comment:
can this line be remove? there is another grid-column-two-thirds within this div